scene/resources/environment.cpp

```diff
@@ -1309,16 +1309,16 @@ Environment::Environment() {
 	set_fog_sun_color(Color(1.0, 0.9, 0.7));
 
 	sdfgi_enabled = false;
-	sdfgi_cascades = SDFGI_CASCADES_6;
-	sdfgi_min_cell_size = 0.2;
+	sdfgi_cascades = SDFGI_CASCADES_4;
+	sdfgi_min_cell_size = 0.3;
 	sdfgi_use_occlusion = false;
 	sdfgi_use_multibounce = false;
-	sdfgi_read_sky_light = false;
+	sdfgi_read_sky_light = true;
 	sdfgi_enhance_ssr = false;
 	sdfgi_energy = 1.0;
 	sdfgi_normal_bias = 1.1;
 	sdfgi_probe_bias = 1.1;
-	sdfgi_y_scale = SDFGI_Y_SCALE_DISABLED;
+	sdfgi_y_scale = SDFGI_Y_SCALE_75_PERCENT;
```

Comment:
Why default to 75%?

Reply:
I discussed this with @reduz. Most level layouts aren't that vertical, and therefore benefit from having a slightly "flattened" probe structure. This should make leaks less noticeable.

We could go as far as defaulting to 50%, but 75% is probably a safer compromise.